corehq/apps/app_manager/app_strings.py

```diff
@@ -207,9 +207,13 @@ def maybe_add_index(text):
                 yield id_strings.custom_assertion_locale(module, form, id), trans(custom_assertion.text)
 
         if hasattr(module, 'case_list_form') and module.case_list_form.form_id:
+            if toggles.FOLLOWUP_FORMS_AS_CASE_LIST_FORM.enabled(app.domain):
+                fallback_name = ugettext("Continue To ") + trans(app.get_form(module.case_list_form.form_id).name)
```

Comment:
Shouldn't this use a format call, so you're translating something like `"Continue to {form_name}"` and the translator can reorder the words if needed?